docs/pull-requests.rst

```diff
@@ -0,0 +1,73 @@
+Preview Documentation from Pull Requests
+========================================
+
+Read the Docs allows to build and preview your documentation from pull/merge requests.
+You can enable it by: 
+
+#. Go to your project dashboard
+#. Go to :guilabel:`Admin > Advanced settings`
+#. Enable the :guilabel:`Build pull requests for this project` option
+#. Click on :guilabel:`Save`
+
+Features
+--------
+
+- **Build on Pull/Merge Request Events:** We create and build a new version when a pull/merge request is open,
+  and when a new commit has been pushed.
+
+- **Warning Banner:** A warning banner is shown at the top of the documentation
+  to let users know that this isn't the main documentation for the project.
+
+  .. note:: This feature is available only for :doc:`Sphinx projects </intro/getting-started-with-sphinx>`.
+
+- **Build Status Report:** When a build is triggered, a build pending notification is send with a link to the build log,
```

Comment:
This should be above the Warning banner, it's a much bigger feature. I think we can probably just remove the warning banner section, since it's not really that useful to document?

Reply:
I think we should document it somewhere, as people some times expect this kind of fetatures to be on mkdocs projects as well. https://github.com/readthedocs/readthedocs.org/issues/7810